docs/workflows/public_data_sharing/retrieve_srr_metadata.md

```diff
@@ -0,0 +1,50 @@
+# Retrieve SRR Metadata Workflow
+
+## Quick Facts
+
+| **Workflow Type** | **Applicable Kingdom** | **Last Known Changes** | **Command-line Compatibility** | **Workflow Level** |
+|---|---|---|---|---|
+| [Public Data Sharing](../../workflows_overview/workflows_type.md/#public-data-sharing) | [Any Taxa](../../workflows_overview/workflows_kingdom.md/#any-taxa) | PHB v2.2.1 | Yes | Sample-level |
+
+## Retrieve SRR Metadata
+
+This workflow retrieves the Sequence Read Archive (SRA) accession (SRR) associated with a given sample accession. It utilizes the `fastq-dl` tool to fetch metadata from SRA and outputs the SRR accession.
+
+### Inputs
+
+<div class="searchable-table" markdown="1">
+
+| **Terra Task Name** | **Variable** | **Type** | **Description**| **Default Value** | **Terra Status** |
+| --- | --- | --- | --- | --- | --- |
+| fetch_srr_metadata | **sample_accession** | String |  SRA-compatible accession, such as a **BioSample ID** (e.g., "SAMN00000000") or **SRA Experiment ID** (e.g., "SRX000000"), used to retrieve SRR metadata. | N/A | Required |
```

Comment:
I would leave the cell blank instead of "N/A" to match the rest of the documentation